lib/apiservers/engine/backends/container_test.go

```diff
@@ -615,7 +616,7 @@ func TestPortInformation(t *testing.T) {
 	mockContainerInfo := &plmodels.ContainerInfo{}
 	mockContainerConfig := &plmodels.ContainerConfig{}
 	containerID := "foo"
-	mockContainerConfig.ContainerID = &containerID
+	mockContainerConfig.ContainerID = containerID
```

Comment:
Well, yes, this one in particular is not problematic. But that doesn't mean the multitude of others is not; its just not easy to go through everyone and understand the ramifications of a particular change. Is the "x-nullable" really necessary right now?